diff --git a/etsi_its_coding/etsi_its_vam_ts_coding/src/vam_ts_ItsPduHeaderVam.c b/etsi_its_coding/etsi_its_vam_ts_coding/src/vam_ts_ItsPduHeaderVam.c
index 49c5a79a..06a665c0 100644
--- a/etsi_its_coding/etsi_its_vam_ts_coding/src/vam_ts_ItsPduHeaderVam.c
+++ b/etsi_its_coding/etsi_its_vam_ts_coding/src/vam_ts_ItsPduHeaderVam.c
@@ -23,7 +23,12 @@ vam_ts_ItsPduHeaderVam_constraint(const asn_TYPE_descriptor_t *td, const void *s
 		/* Nothing is here. See below */
 	}
 	
-	return td->encoding_constraints.general_constraints(td, sptr, ctfailcb, app_key);
+	/* prevent infinite recursion */
+	if (td->encoding_constraints.general_constraints != vam_ts_ItsPduHeaderVam_constraint) {
+		return td->encoding_constraints.general_constraints(td, sptr, ctfailcb, app_key);
+	} else {
+		return 0;
+	}
 }
 
 /*
