diff --git a/TS103561_LUKAS_MCM.asn b/TS103561_LUKAS_MCM.asn
index 005491e..194d022 100644
--- a/TS103561_LUKAS_MCM.asn
+++ b/TS103561_LUKAS_MCM.asn
@@ -5,7 +5,7 @@ DEFINITIONS AUTOMATIC TAGS ::=
 BEGIN
 IMPORTS
     ItsPduHeader,
-    StationID,
+    StationId,
     HeadingValue,
     CartesianCoordinateLarge,
     Speed,
@@ -110,7 +110,7 @@ AdviceUpdateID ::= INTEGER (0..65535)
 ManeuverResponse ::= ENUMERATED {accept(0), reject(1), pending(2)}
 
 SuggestedManeuverContainer ::= SEQUENCE {
-	targetStationID     StationID,
+	targetStationID     StationId,
 	suggestedManeuver   SuggestedManeuver OPTIONAL
 }
 
@@ -122,7 +122,7 @@ SuggestedManeuver ::= SEQUENCE {
 	maneuverParameters                  ManeuverParameters
 }
 
-ParticipatingRoadUserIDContainer ::= SEQUENCE SIZE(1..128, ...) OF StationID
+ParticipatingRoadUserIDContainer ::= SEQUENCE SIZE(1..128, ...) OF StationId
 
 ConfirmationRequiredFlag ::= BOOLEAN
 
